src/Plugin/media/Source/Embeddable.php

```diff
@@ -0,0 +1,155 @@
+<?php
+
+namespace Drupal\stanford_media\Plugin\media\Source;
+
+use Drupal\media\Plugin\media\Source\OEmbed;
+use Drupal\media\MediaInterface;
+use Drupal\media\MediaTypeInterface;
+
+/**
+ * @MediaSource(
+ *   id = "embeddable",
+ *   label = @Translation("Stanford Embedded Media"),
+ *   description = @Translation("Embeds a third-party resource."),
+ *   default_thumbnail_filename = "generic.png",
+ *   providers = {"ArcGIS StoryMaps", "CircuitLab", "Codepen", "Dailymotion", "Facebook", "Flickr", "Getty Images", "Instagram", "Issuu", "Livestream", "MathEmbed", "Simplecast", "SlideShare", "SoundCloud", "Spotify", "Stanford Digital Repository", "Twitter"},
```

Comment:
This should not be hard coded. it should pull from the settings in the oembed module.

Reply:
it actually must be hardcoded:

"The providers key is crucial -- it defines which oEmbed providers (i.e., third-party services) will be allowed to be used by this plugin, and the plugin must explicitly opt in to each provider. The canonical list of providers is available at https://oembed.com/providers.json. The supported_providers array must contain the names of the providers you want to support, exactly as they appear in the canonical provider list."

per: drupal.org/node/2966029

Reply:
Should we provide an alter in this module then to support the rest?

Reply:
> Should we provide an alter in this module then to support the rest?

I've been thinking about how to do that, because as it turns out there are oEmbed providers who aren't on the canonical list, and there are providers on the canonical list which are misconfigured there, and thus don't actually work.  Ishi found that codepen was one such provider.  But I don't have a good way of approaching that yet.